lib/infobox/node.js

```diff
@@ -189,6 +189,30 @@ define(['chroma-js', 'moment', 'tablesort', 'helper'],
 
     return function (config, el, router, d) {
       var linkScale = chroma.scale(chroma.bezier(['#04C714', '#FF5500', '#F02311'])).domain([1, 5]);
+      function renderNeighbourRow(n) {
+        var icons = [];
+        var name = [];
+        var unknown = !(n.node);
+
+        icons.push(V.h('span', { className: n.incoming ? 'ion-ios-arrow-thin-left' : 'ion-ios-arrow-thin-right' }));
+        if (!unknown && helper.hasLocation(n.node)) {
+          icons.push(V.h('span', { className: 'ion-location' }));
+        }
+
+        if (!unknown) {
+          name.push(V.h('a', { href: router.getUrl({ n: n.node.nodeinfo.node_id }), onclick: router.node(n.node), className: 'online' }, n.node.nodeinfo.hostname));
+        } else {
+          name.push(n.link.id);
+        }
+
+        var td1 = V.h('td', icons);
+        var td2 = V.h('td', name);
+        var td3 = V.h('td', { style: { color: linkScale(n.link.tq).hex() } }, helper.showTq(n.link));
```

Comment:
Needs testing for gateway, node visible and other stuff . maybe in a separate PR. Also linkcolor and stuff..